pynicotine/slskmessages.py

```diff
@@ -1700,15 +1700,15 @@ def parseNetworkMessage(self, message):
 
     def makeNetworkMessage(self):
         if self.pic is not None:
-            pic = chr(1) + self.packObject(self.pic)
+            pic = bytes(1) + self.packObject(self.pic)
```

Comment:
The argument to `bytes()`, if it is an integer, is interpreted as the length of the `bytes` object, not its contents (https://docs.python.org/3/library/functions.html#func-bytes -> https://docs.python.org/3/library/stdtypes.html#bytearray)  This does not create a bytes object of 1 byte with value 1, but a byte object of length 1 with value 0. Similarly, `bytes(0)` creates an empty `bytes`. The correct way to do this is `bytes([1])` or `bytes((1,))`.

Reply:
Of course you're right, shame on me. Fixed.